xnmt/decoder.py

```diff
@@ -116,7 +120,9 @@ def calc_loss(self, context, ref_action):
       return dy.pickneglogsoftmax(scores, ref_action)
```

Comment:
This needs to be implemented in non-minibatch mode as well.

Reply:
done via the latest commit !